examples/dmd/wave_equation.cpp

```diff
@@ -24,6 +24,19 @@
 // Elapsed time for solving FOM: 3.122185e+00 second
 // Elapsed time for training DMD: 6.904051e-01 second
 // Elapsed time for predicting DMD: 2.496171e-03 second
+//
+// Sample run using snapshotDMD and using projected initial conditions for the
+//  predicition phase.
+//
+// Command 2:
+//  wave_equation -tf 2 -nwinsamp 25 -rdim 27 -visit -snap -proj
+//
+// Output 2:
+//  Relative error of DMD solution (u) at t_final: 2 is 0.0029577152
```

Comment:
I realize that the error is higher than the original outputs, with a mixed effect of enabling snapshot interpolation and the projection of initial condition. It could be helpful if we know which one is the source of the increase of the error.

Reply:
I have updated the output for command 1 to reflect what I obtain on quartz.  I get a different error than what was reported previously while running on this branch, and confirmed the same (new) numbers on the master branch.  Additionally, I have added two new examples: the first performs the current "command 1" with projection enabled, and the second performs the current "command 2" with projection disabled.  The errors are much closer together and should, hopefully, clear things up.